src/world/message_server.cpp

```diff
@@ -43,18 +44,20 @@ struct zone_settings_t
     uint32 misc   = 0;
 };
 
+using InternalHandler = std::function<void(std::vector<uint8>&&, in_addr, uint16)>;
```

Comment:
This is a different signature to whats used inside message handler, should they be the same?

Reply:
good catch. It's not. I'm not sure how it compiles / works.